dashboard/app/controllers/api/v1/regional_partners_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::RegionalPartnersController < ApplicationController
+  # GET /api/v1/regional-partner/<school_district_id>
+  def index
+    school_district = SchoolDistrict.find(params[:school_district_id])
+    render json: school_district.regional_partner
```

Comment:
What if it's passed an invalid id? I think now it will blow up with `undefined method regional_partner on nil:NilClass` (500 error). How about rendering a 404?